Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -63,19 +63,28 @@ void CWII_IPC_HLE_Device_sdio_slot0::OpenInternal()
 {
 	const std::string filename = File::GetUserPath(D_WIIROOT_IDX) + "/sd.raw";
 	m_Card.Open(filename, "r+b");
+	if (m_Card)
+		return;
+	
+	WARN_LOG(WII_IPC_SD, "Failed to open SD Card image, trying read-only access...");
+	m_Card.Open(filename, "rb");
+	if (m_Card)
+	{
+		SConfig::GetInstance().bEnableMemcardSdWriting = false;
```

Comment:
Ah, you're right. I did a search for bEnableMemcardSdWriting but missed the GC in the header.

I'm starting to see how a separate flag could be better. The real Wii supports write-protected SD cards. Even without flagging bEnableMemcardSdWriting, everything, as you said, would work. Only, triggering the SD card writes would require you to restart emulation. There's no on-the-fly support like on a Wii real with a real SD card with a write-protect toggle.